far/FarEng.hlf.m4

```diff
@@ -5846,7 +5858,7 @@ $ #far:config System.CopySecurityOptions#
     Параметр "System.CopySecurityOptions" не влияет на создание связей (Alt+F6). В этом случае 
 права всегда копируются.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+    Изменение этого параметра возможно через ~far:config~@FarConfig@
```

Comment:
This line appears about 60 times in the English help, in all untranslated `far:config` topics. I did not change them to avoid language mix inside single topic. If you think it is still better to change them all even before translating the topics themselves, please let me know. I am not strong on this.

Reply:
No need to change all of them.
I mentioned this one only because you already touched it.
It's ok to leave as is for now.

Reply:
> I mentioned this one only because you already touched it.

I understand, and I actually thought about translating the line when I touched it. However, I finally decided to keep the topics in the single language. I'll keep them in Russian.